Gems/ExternalProfilers/SuperluminalProfiler/Code/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(SUPERLUMINAL_API_PATH "C:/Program Files/Superluminal/Performance/API" CACHE PATH "Path to the Superluminal 3rd party profiler API folder.")
```

Comment:
Having hardcoded path here does not seem right. What do other gems do?

Reply:
Hey ! Its just the default value and is exposed in the cmake during configure